sensors/01-meteorological-data.Rmd

```diff
@@ -144,88 +127,63 @@ The data represent 5 minute summaries aggregated from 1/s observations.
 
 #### Using Curl
 
-First, this is what the API looks like as a URL. Try pasting it into your browser
+First, below is what the API looks like as a URL. Try pasting it into your browser.
 
-https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-02&until=2017-01-31
+https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2018-04-01&until=2018-08-01
 
-This is how you can automatically download the met data to a local file:
+These data are for one stream within a chosen time period. They can be 
+automatically downloaded into a file on your computer by typing the following
+into the command line. This uses the URL from above and the new file is named
+spectra.json.
 
 ```{sh eval=FALSE}
-curl -O spectra.json -X GET https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-02&until=2017-01-31
+curl -o spectra.json -X GET https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2018-04-01&until=2018-08-01
 ```
 
-And this is how you can access the data in R:
+#### Using R
+
+The following code sets the defaults for showing R code. 
+```{r met-setup}
+knitr::opts_chunk$set(cache = FALSE, message = FALSE)
+```
 
+And this is how you can access the same data in R. This uses the jsonlite R package 
+and desired URL to pull the data in. The data is in a dataframe with two nested
+dataframes, called `properties` and `geometries`. 
 
 ```{r met-geostream}
 library(dplyr)
 library(ggplot2)
 library(jsonlite)
+library(lubridate)
 
-mac_weather.list <- jsonlite::fromJSON('https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2017-01-02&until=2017-01-31', flatten = FALSE)
-
-# change time to human-readable
-mac_weather <- mac_weather.list$properties %>% 
-  mutate(time = lubridate::ymd_hms(mac_weather.list$end_time))
+weather_all <- fromJSON('https://terraref.ncsa.illinois.edu/clowder/api/geostreams/datapoints?stream_id=46431&since=2018-04-01&until=2018-08-01', flatten = FALSE)
 ```
 
-### Using 
+The `geometries` dataframe is then pulled out from these data, which contains
+the datapoints from this stream. This is combined with a transformed version of the
+end of the time period from the stream. 
 
-## Weather Summary
+```{r met-datapoints}
+weather_data <- weather_all$properties %>% 
+  mutate(time = ymd_hms(weather_all$end_time))
+```
 
+### Weather Plots
 
-#### Wind Speed
+Create overlaid time series plots for two of the eight variables, wind speed and 
+precipitation rate, in the newly created dataframe. 
 
 ```{r weather}
 theme_set(ggthemes::theme_few())
-ggplot(data = mac_weather) +
-  geom_point(aes(x = time, y = wind_speed), size = 0.1)
-```
-
-#### Rainfall
-
-```{r precipitation}
-ggplot(data = mac_weather) +
-  geom_point(aes(x = time, y = precipitation_rate), size = 0.1)
+ggplot(data = weather_data) +
```

Comment:
please use a separate plot for each variable. For example: 

![image](https://user-images.githubusercontent.com/464871/51920955-a9632800-23a3-11e9-9131-a4c80cf3fbfc.png)

 see: https://rpubs.com/dlebauer/ciat-met

Reply:
Oh, when you said one plot I for some reason thought you meant both variables on the same plot. So you want one plot for just one of the variables?

Reply:
I thought you said you only wanted one plot, of either precipitation or wind speed?